erpnext/accounts/utils.py

```
@@ -2724,3 +2724,121 @@ def build_qb_match_conditions(doctype, user=None) -> list:
+	if cfg.get("check_credit_limit"):
+		_check_credit_limit_warn(doc)
+
+	if cfg.get("check_packed_qty"):
+		_check_packed_qty_warn(doc)
+
+
+def _check_prev_docstatus(doc):
+	try:
+		if doc.get("check_prev_docstatus"):
+			doc.check_prev_docstatus()
+	except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)